### PR TITLE
chore: optionality card component and test LESQ-130

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

- Optionality Card Component Created
- UnitList test file updated
- Optionality fixture
- Added calc to Flex styles

## Issue(s)

Fixes # LESQ-130

## How to test

1. Go to https://deploy-preview-1793--oak-web-application.netlify.thenational.academy
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):

Screenshot 2023-08-15 at 17 47 19
Screenshot 2023-08-15 at 17 46 54

Screenshot 2023-08-15 at 17 48 04

How it should now look
Screenshot 2023-08-15 at 17 45 51
Screenshot 2023-08-15 at 17 44 47

Screenshot 2023-08-15 at 17 48 34

## Checklist

- [x] Added or updated tests where appropriate
- [x] Manually tested across browsers / devices
- [x] Considered impact on accessibility
- [x] Design sign-off
- [x] Approved by product owner

### CODE REVIEW

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
- Could this be sentence case (so "Unit options")
Screenshot 2023-08-16 at 09 00 24
- Looks like there is too much space between the rows
Screenshot 2023-08-16 at 09 08 01

**Comment by k-huggs** [comment]
> 

Should both be fixed ✅

**Comment by Monika-Koziol** [comment]
@k-huggs looking good to me.

**Review by Monika-Koziol — APPROVED** [review]
Can't comment on the code but the designs are looking good.

File: src/components/OptionalityCard/OptionalityCard.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          $pb={12}
+          $width={"100%"}
+        >
+          <Flex $mb={16}>
+            <ListTitle index={index}>{unitTitle}</ListTitle>
+          </Flex>
+          <Flex $cg={12} $width={"100%"} $flexWrap={"wrap"}>
+            {unitOptions.map((unitOption: UnitOption) => {
+              return (
+                <Flex $width={["100%", "calc(50% - 6px)"]}>
```
Comment (JBR90) [inline src/components/OptionalityCard/OptionalityCard.tsx:64]: Can you add a key here?

**Review by JBR90 — COMMENTED** [review]
Looks great, one very small comment

**Review by JBR90 — APPROVED** [review]

**Review by JBR90 — APPROVED** [review]

File: src/components/OptionalityCard/OptionalityCard.tsx (hunk 1 above)
Comment (k-huggs) [reply src/components/OptionalityCard/OptionalityCard.tsx]: added ✅

**Pull request merged**